src/sentry/testutils/silo.py

```diff
@@ -85,22 +83,25 @@ def strip_silo_mode_test_suffix(name: str) -> str:
 class SiloModeTestDecorator:
     """Decorate a test case that is expected to work in a given silo mode.
 
-    Tests marked to work in monolith mode are always executed.
-    Tests marked additionally to work in region or control mode only do so when the test is marked as stable=True
+    A test marked with a single silo mode runs only in that mode by default. An
+    `include_monolith_run=True` will add a secondary run in monolith mode.
 
-    When testing in a silo mode, if the decorator is on a test case class,
+    If a test is marked with both control and region modes, then the primary run will
+    be in monolith mode and a secondary run will be generated in each silo mode.
+
+    When testing on more than one mode, if the decorator is on a test case class,
     an additional class is dynamically generated and added to the module for Pytest
     to pick up. For example, if you write
 
     ```
-        @control_silo_test
+        @control_silo_test(include_monolith_run=True)
```

Comment:
For the relocation tests, we want to test in two configurations: Silo mode-only for the default `backend test`, and then using `Monolith` mode AND a single monolith DB for the `monolith-dbs-test` runner. The former tests relocation to/from SaaS, the latter tests relocation to/from self-hosted, which will remain monolithic in all ways for the foreseeable future. Using a monolith DB for testing is particularly important, as we want to check that a single atomic import transaction remains possible for self-hosted users.

I see that in this PR you are changing how the decorators work, and removing `SENTRY_USE_MONOLITH_DBS`. Will the above testing regime still work once we do that?

Reply:
Per discussion, we will continue to support `make test-monolith-dbs`.